pkg/master/master_test.go

```diff
@@ -370,114 +393,109 @@ func TestAPIVersionOfDiscoveryEndpoints(t *testing.T) {
 }
 
 func TestDiscoveryAtAPIS(t *testing.T) {
```

Comment:
I address the TODO (I think it's left silently by etune months ago :). 

Github's diff is misleading, the only change is I called newLimitedMaster() instead of newMaster().